lib/kvdb/kvdb_rparams.c

```diff
@@ -328,15 +313,18 @@ mclass_policies_converter(const struct param_spec *ps, const cJSON *node, void *
                     }
                 }
                 if (dtype == -1) {
-                    log_err("Invalid media class policy data type: %s, must be one of key or value",
-                            dtype_json->string);
+                    log_err(
+                        "Invalid media class policy data type: %s, must be one of key or value",
+                        dtype_json->string);
                     return false;
                 }
 
                 const int sz = cJSON_GetArraySize(dtype_json);
-                if (sz > HSE_MPOLICY_MEDIA_CNT) {
-                    log_err("Too many items in media class policy data type array (max = %d)",
-                            HSE_MPOLICY_MEDIA_CNT);
+                if (sz > MP_MED_COUNT) {
+                    log_err(
+                        "Too many items in media class policy data "
+                        "type array (max = %d)",
+                        MP_MED_COUNT);
```

Comment:
Is there a reason to wrap these strings above?

Reply:
You will hate my answer...Happy to revert the changes if you want since in theory they shouldn't be a part of this commit anyway.